specifications/sep-payment-channel-transactions.md

```diff
@@ -229,45 +226,58 @@ payments to one another within the channel.
 For example, if the channel initial state is $30 of which $10 belongs to I and
 $20 belongs to R, the first closing transaction will disburse $10 to I and $20
 to R. If I makes a payment of $2 to R, this update process will involve I and R
-agreeing on a new declaration and aclosing transaction that supersedes all
-previous declaration and closing transaction and that will disburse $8 to I and
-$22 to R.
+agreeing on new declaration, payment, and closing transactions that supersede
+all previous declaration, payment, and closing transaction and that will
+disburse $8 to I and $22 to R.
 
 To update the payment channel state, the participants:
 
 1. Increment i.
 2. Sign and exchange a closing transaction C_i.
-3. Sign and exchange a declaration transaction D_i.
+3. Sign and exchange zero or more payment transactions P_{i,a}.
+4. Sign and exchange a declaration transaction D_i.
 
-It is important that D_i is signed after C_i because D_i will invalidate any
-previously signed C_i. If I and R signed and exchanged D_i first either party
-could prevent the channel from closing without coordination by submitting D_i
-and refusing to sign C_i.  The participants would not be able to close the
-channel, or regain control of the accounts, and the assets within without
-coordinating with each other.
+It is important that D_i is signed after C_i and P_i because D_i will invalidate
+any previously signed C_i and P_i. If I and R signed and exchanged D_i first
+either party could prevent the channel from closing without coordination by
+submitting D_i and refusing to sign C_i.  The participants would not be able to
+close the channel, or regain control of the accounts, and the assets within
+without coordinating with each other.
 
 The transactions are constructed as follows:
 
-- C_i, the _closing transaction_, disburses funds from EI to ER and/or from ER
-to EI, and changes the signing weights on EI such that I unilaterally controls
-EI, and the signing weights on ER such that R unilaterally controls ER.  C_i has
-source account EI, sequence number s_i+1, a `minSeqAge` of O (the observation
-period time duration), and a `minSeqLedgerGap` of O (the observation period
-ledger count).
+- C_i, the _closing transaction_, changes the signing weights on EI such that I
+unilaterally controls EI, and the signing weights on ER such that R unilaterally
+controls ER.  C_i has source account EI, sequence number s_i+1+n.
 
   The `minSeqAge` and `minSeqLedgerGap` prevents a misbehaving party from
   executing C_i when the channel state has already progressed to a later
-  iteration number, as the other party has the period O to invalidate C_i by
+  iteration number, as the other party has the period 2O to invalidate C_i by
```

Comment:
2O is hard to distinguish from 20 (twenty) when reading:
```suggestion
  iteration number, as the other party has the period 2*O to invalidate C_i by
```

Reply:
Oops, 2O should be gone, it should be O. 2O was required in an earlier commit on this PR, then I removed it. Fix incoming.

Reply:
I've fixed this in 9b78df3.

Although I realize this creates a new issue, if there are no payments, the close is not delayed. I'll fix that shortly.

Reply:
I fixed the second issue with the observation period missing on close with no payments in 9fa8ec1.

Reply:
cool, thanks for changing